src/_guides/language/effective-dart/usage.md

```diff
@@ -119,6 +119,60 @@ library. Follow these two rules:
 * An import path should never contain `/lib/`.
 * A library under `lib` should never use `../` to escape the `lib` directory.
 
+## Booleans
+
+### DO use `??` to convert `null` to a Boolean value.
+
+You sometimes have an expression that can evaluate to `true`, `false`, or `null`
+and you need to pass the result to something that doesn't accept `null`. A
+common case is the result of a null-aware method call being used as a condition:
+
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (null-aware-condition)"?>
+{% prettify dart %}
+if (optionalThing?.isEnabled) {
+  print("Have enabled thing.");
+}
+{% endprettify %}
+
+This code will throw an exception if `optionalThing` is `null`. To fix this, you
+need to "convert" the `null` value either to `true` or `false`. There are two
+ways to do this, using `??` or `==`:
```

Comment:
If you read this without going further down, it sounds like either one is OK. Let's be more judgy here. Maybe:

Although you could do this using `==`, we recommend using `??`: